console/billing/additional-content/understanding-savings-plans.mdx

```diff
@@ -101,19 +101,13 @@ There is currently one available Savings Plan type: the Compute Savings Plan.
 The **Compute Savings Plan** can be used with the following resources, simultaneously and across all regions:
 
     - Instances 
-      - Cost-optimized (COPARM1, PLAY2, PRO2)
+      - Cost-optimized (DEV1, GP1, COPARM1, PLAY2, PRO2)
       - Production optimized (ENT1, POP2)
       - Workload optmized (POP2 HC) 
-    - Dynamic IPs, IPv6, Reserved IPs
-    - Block Storage SSD
-    - Block Storage volumes
-    - Block Storage Low Latency volumes
 
 The following resources are **not** covered by the Savings Plan discount:
 
-    - Instances H100, RENDER, L40, L4, DEV1, GP1, START1, X64, POP-WIN
-    - Instance and Block Storage snapshots 
-    - Local storage 
+    - Instances H100, RENDER, L40, L4, COPARM1, DEV1, GP1, START1, X64, POP-WIN
```

Comment:
Instances H100, RENDER, L40, L4, COPARM1, START1, X64, POP-WIN
- DEV1, GP1 should be removed